### PR TITLE
Some cleanups to variable naming and code organization

### PR DESCRIPTION
**Pull request by snario**

### CODE REVIEW

File: packages/node/src/network-configuration.ts [hunk 1]
```diff
@@ -2,60 +2,71 @@ import RopstenContracts from "@counterfactual/contracts/networks/3.json";
 import RinkebyContracts from "@counterfactual/contracts/networks/4.json";
 import KovanContracts from "@counterfactual/contracts/networks/42.json";
 import {
-  ContractMigration,
-  NetworkContext,
-  networkContextProps
+  DeployedContractNetworksFileEntry,
+  NAMES_OF_DEPLOYED_CONTRACTS_REQUIRED_FOR_COUNTERFACTUAL,
```
Comment (IIIIllllIIIIllllIIIIllllIIIIllllIIIIll) [inline packages/node/src/network-configuration.ts]: The names networkContext and networkContextProps should be kept in sync

Comment (IIIIllllIIIIllllIIIIllllIIIIllllIIIIll) [reply packages/node/src/network-configuration.ts]: why is it changed to `EXPECTED_CONTRACT_NAMES...`? What's wrong with `networkContextProps`

File: packages/node/test/unit/install.spec.ts [hunk 2]
```diff
@@ -24,18 +28,28 @@ import {
   FreeBalanceStateJSON
 } from "../../src/models/free-balance";
 import { Store } from "../../src/store";
-import { EMPTY_NETWORK } from "../integration/utils";
 import { MemoryStoreService } from "../services/memory-store-service";
 
 import { createAppInstanceProposalForTest } from "./utils";
 
+const NETWORK_CONTEXT_OF_ALL_ZERO_ADDRESSES = EXPECTED_CONTRACT_NAMES_IN_NETWORK_CONTEXT.reduce(
```
Comment (cf19drofxots) [inline packages/node/test/unit/install.spec.ts]: This should be in a more generic place as it's not specific to this test suite.

File: packages/node/src/node.ts [hunk 3]
```diff
@@ -79,24 +79,19 @@ export class Node {
...
-        this.blocksNeededForConfirmation = blocksNeededForConfirmation;
-      }
+    if (typeof networkContext === "string") {
+      this.networkContext = getNetworkContextForNetworkName(networkContext);
     } else {
-      // Used for testing / ganache
       this.networkContext = networkContext;
     }
+
+    this.blocksNeededForConfirmation = blocksNeededForConfirmation;
```
Comment (cf19drofxots) [inline packages/node/src/node.ts]: Remove line, this isn't doing anything

File: packages/node/test/unit/install.spec.ts (hunk 2 above)
Comment (snario) [reply packages/node/test/unit/install.spec.ts]: It actually is specific to this test suite

Comment (cf19drofxots) [reply packages/node/test/unit/install.spec.ts]: How? It's a network context object with all zero addresses. I don't see how it's specific to this test suite.

Comment (snario) [reply packages/node/test/unit/install.spec.ts]: It is never used anywhere else

Comment (cf19drofxots) [reply packages/node/test/unit/install.spec.ts]: That's not a good reason to place a generic object in a test-specific context, but I digress.

**Pull request merged**